tests/test_service.py

```diff
@@ -62,7 +62,7 @@ def tearDown(self):
         db.session.remove()
         db.drop_all()
 
-    def _create_wishlists(self, count=1):
+    def _create_wishlists(self, count):
```

Comment:
I think we need to have count = 1. This just sets the default value of count to 1. Makes the function easier to use.

Reply:
How could we assume when developers want to use this helper function, they only need to create one wishlist by default? Note that the naming of this function is _create_wishlist**s**, not _create_wishlist.